core/src/impl/Kokkos_Core.cpp

```diff
@@ -532,6 +532,57 @@ void initialize(int& narg, char* arg[])
       iarg++;
     }
 
+    //Read environment variables
+    auto env_num_threads_str = std::getenv("KOKKOS_NUM_THREADS");
+    if (env_num_threads_str) {
```

Comment:
Prefer to write `if (env_num_threads_str == nullptr)`.

Reply:
@mhoemmen . Thanks for your comments.